server/db/postSetup.sql

```diff
@@ -0,0 +1,15 @@
+DROP TABLE IF EXISTS post;
+
+-- CREATE TABLE post (
+--     id serial PRIMARY KEY,
+--     title varchar(255) NOT NULL,
+--     user varchar(100),
+--     body varchar(400) 
+-- );
+
+CREATE TABLE post(
+    id serial PRIMARY KEY,
+    title varchar(255) NOT NULL,
+    username varchar(100),
+    body varchar(500)
```

Comment:
An improvement over varchar would possibly be the text data type in postgres, as it functions like varchar, and has no character limitation.